examples/healthData/src/healthDataSchema.ts

```diff
@@ -0,0 +1,67 @@
+// The following is a schema definition for enetring health data.
+
+export interface HealthDataResponse {
+    // Return this if JSON has ALL required information. Else ask questions
+    data?: HealthData;
+    // Use this to ask questions and give pertinent responses
+    message?: string;
+    // Use this parts of the user request not translateed, off topic, etc
+    notTranslated?: string;
+}
+
+export interface HealthData {
+    medication?: Medication[];
+    condition?: Condition[];
+    other?: OtherHealthData[];
+}
+
+// Meds, pills etc.
+export interface Medication {
+    // Fix any spelling mistakes, especially phonetic spelling
+    name: string;
+    // E.g. 2 tablets, 1 cup. Required
+    dose: ApproxQuantity;
+    // E.g. twice a day. Required
+    frequency: ApproxQuantity;
+    // E.g. 50 mg. Required
+    strength: ApproxQuantity;
+}
+
+// Disease, Ailment, Injury, Sickness
+export interface Condition {
+    // Fix any spelling mistakes, especially phonetic spelling
+    name: string;
+    // When the condition started? Required
+    startDate: ApproxDatetime;
+    // Always ask for current status of the condition
+    status: "active" | "recurrence" | "relapse" | "inactive" | "remission" | "resolved" | "unknown";
+    // If the condition was no longer active
+    endDate?: ApproxDatetime;
+}
+
+// Use for health data that match nothing else. E.g. immunization, blood prssure etc
+export interface OtherHealthData {
+    text: string;
+    when?: ApproxDatetime;
+}
+
+export interface ApproxQuantity {
+    // Default: Unknown. Required
+    displayText: string;
+    // Optional: only if precise quantities are available
+    quantity?: Quantity;
+}
+
+export interface ApproxDatetime {
+    // Default: Unknown. Required
```

Comment:
```suggestion
    // Default: "unknown"
```

Reply:
What does it mean if there's a default but it's required? Why does the LLM have to fill it in?

Reply:
The field is filled as unknown by the LLM.

Reply:
Kind of weird - seems like it would be better off as optional entirely in that case

Reply:
I agree but in this we want the field. Anyways its consistent across both TS and Python.